ui/components/app/snaps/snap-authorship-expanded/snap-authorship-expanded.js

```diff
@@ -53,6 +61,24 @@ const SnapAuthorshipExpanded = ({ snapId, className, snap }) => {
   const subjectMetadata = useSelector((state) =>
     getTargetSubjectMetadata(state, snapId),
   );
+  const snapRegistryData = useSelector((state) =>
+    getSnapRegistryData(state, snapId),
+  );
+  const { website = undefined } = snapRegistryData?.metadata ?? {};
+  const [safeWebsite, setSafeWebsite] = useState('');
```

Comment:
Should this be `null` or `undefined` by default?

Reply:
Updated to use `null` now.